src/main/java/bio/terra/janitor/service/janitor/JanitorService.java

```diff
@@ -24,6 +24,12 @@ public JanitorService(JanitorDao janitorDao) {
 
   public CreatedResource createResource(CreateResourceRequestBody body) {
     Instant creationTime = Instant.now();
+    return createResourceWithCreation(body, creationTime);
+  }
+
+  /** Creates resources with given creation time. */
+  public CreatedResource createResourceWithCreation(
+      CreateResourceRequestBody body, Instant creationTime) {
```

Comment:
I think it's simpler to include the creation time within the CreateResourceRequetBody. Let's do that, in this PR or the next.

Reply:
okay, I will do it in next PR